src/main/python/ayab/signal_receiver.py

```diff
@@ -36,13 +40,13 @@ class SignalReceiver(QObject):
     # signals are defined as class attributes which are
     # over-ridden by instance attributes with the same name
     start_row_updater = Signal(int)
-    progress_bar_updater = Signal(int, int, int, 'QString')
```

Comment:
turns out we can just use `str` for all of these since PySide doesn't _have_ a `QString` object.